src/OpenTelemetry.AutoInstrumentation/AutoInstrumentationEventSource.cs

```diff
@@ -0,0 +1,46 @@
+// <copyright file="AutoInstrumentationEventSource.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics.Tracing;
+
+namespace OpenTelemetry.AutoInstrumentation;
+
+/// <summary>
+/// EventSource implementation for OpenTelemetry AutoInstrumentation implementation.
+/// </summary>
+[EventSource(Name = "OpenTelemetry-AutoInstrumentation")]
+internal class AutoInstrumentationEventSource : EventSource
+{
+#pragma warning disable SA1401 // Fields should be private
+    public static AutoInstrumentationEventSource Log = new AutoInstrumentationEventSource();
+#pragma warning restore SA1401 // Fields should be private
+
+    /// <summary>Logs as Trace level message.</summary>
```

Comment:
It can be a dummy question, but Tracer level is the same as Informational (two lines later)?

Reply:
Yes, that is correct. We have the same implementation here - https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/src/OpenTelemetry.AutoInstrumentation.StartupHook/StartupHookEventSource.cs